python/tidy/servo_tidy/tidy.py

```diff
@@ -942,6 +942,20 @@ def run_lint_scripts(only_changed_files=False, progress=True):
             yield error
 
 
+def check_commits(path='.'):
+    """Gets all commits since the last merge."""
+    args = ['git', 'log', '-n1', '--merges', '--format=%H']
+    last_merge = subprocess.check_output(args, cwd=path).strip()
+    args = ['git', 'log', '{}..HEAD'.format(last_merge), '--format=\'%H %s\'']
```

Comment:
Oh right, that's a remnant of when I also returned which commit was wrong, before learning that tidy generally doesn't really return much detail.